### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by mostbrilliant1** [bcssov/IronyModManager#170] [General]
Hello friends,

I just wanted to give my thanks for making this mod. It is unreal, and it is something to be proud of.

Thanks again.

**Comment by Queenofdragons6** [comment]
I second this. It's so useful and convenient, I wish I could use it for non-Paradox games.

**Comment by urusaich** [comment]
My thanks to the authors. Ivory mod manager fixes a broken mod and makes it work!
